app/views/items/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for [user, item] do |f| %>
+  <div class="form-group">
+    <%= f.label :description %>
+    <%= f.text_field :description, class: 'form-control', placeholder: "Enter the task" %>
```

Comment:
you can create a new migration file that changes the name of the column.  the method is:

`rename_column :table, :old_column, :new_column`